index.bs

```diff
@@ -737,6 +769,51 @@ authorizing an authenticator.
 </div>
 
 
+#### Construct the credential - `constructCredentialCallback` method #### {#constructCredentialCallback}
+
+This method's purpose is to map |credentialCreationData|'s [=items=] into a new {{PublicKeyCredential}} object associated
+with the caller's [=environment settings object/global object=], while running on the main [=event loop=]. See also
+[[!CREDENTIAL-MANAGEMENT-1]].
+
+This algorithm accepts one argument:
+
+    :   |global|
+    ::  Which must be its caller's [=current settings object=]'s [=environment settings object/global object=].
+
+When this method is invoked, the user agent MUST execute the following algorithm:
+
+1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
+    bytes of |credentialCreationData|.<code>[=attestationObjectResult=]</code>'s value.
+
+3.  Let |id| be |attestationObject|.<code>authData.attestation data.credential ID</code> (see [[#sec-attestation-data]]
```

Comment:
Fun that our field names have spaces in them.

Reply:
yeah, that's because they are informal on-the-fly names, which need to be formalized, pls see issue #233 subissues (2) & (3)   ("formalized" whether by adding a "name" column to the tables, or by using ABNF, or whatever)